doc/source/install.rst

```diff
@@ -299,5 +303,5 @@ Optional Dependencies
 
    Without the optional dependencies, many useful features will not
    work. Hence, it is highly recommended that you install these. A packaged
-   distribution like `Anaconda <http://docs.continuum.io/anaconda/>`__, or `Enthought Canopy
+   distribution like `Anaconda <http://docs.continuum.io/anaconda/>`__, `ActivePython <https://www.activestate.com/activepython/downloads>`__, or `Enthought Canopy
```

Comment:
I can't find any information about ActivePython being a distribution that included pandas and its dependencies, so I find it a bit strange to list it here. Eg http://docs.activestate.com/activepython/3.6/pkg/ does not list it.

Reply:
@jreback If you prefer it to say something else, let me know. That is the page where they've requested we list the ActivePython distribution.

Reply:
I am fine with it being listed as a way to install python (the other edits), but just as far as I can see (I don't know it, I just go from what I find on their website) it is not a distribution that includes pandas?

Reply:
this place is fine. I tihnk @jorisvandenbossche is just asking whether activestate is like 'Anaconda' in that it *includes* pandas (I think no?)

Reply:
I don't actually know the answer to that question.

Reply:
If we don't know, we should ask somebody who knows. 
As I said, I don't necessarily have a problem with mentioning ActivePython as a way to install python/pandas, but the instructions we provide should be correct! (otherwise it will only be confusing for users and turn them away from ActivePython)

Can't somebody from ActivePython weigh in here?

Reply:
I've asked them to weigh in.

Reply:
Hi @Dr-G,

Thanks for the opportunity to clarify.  Our distributions for Python 2.7.14 and 3.5.4 include Pandas, its dependencies, and many of the optional dependencies. @jorisvandenbossche is correct that our Python 3.6 distribution doesn't currently include Pandas. We will be updating the package list to match 3.5.4 at some point in the next few months. We're just starting to see demand for 3.6 from our enterprise customers. 

For now, could the last sentence read something similar to the following: 
_A packaged distribution like Anaconda, Enthought Canopy, or ActivePython (version 2.7 or 3.5) may be worth considering._

Reply:
@jreback are we all set with this? Let me know if you need me to do anything/make any changes. Thanks!

Reply:
@davetlewis-van Thanks for the clarification. That explains why I couldn't find it with for python 3.6.

Reply:
I added the version clarification in https://github.com/pandas-dev/pandas/pull/19964. @davetlewis-van Once version 3.6 is updated to include pandas, feel free to do a PR to update the instructions.